modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -3,33 +3,33 @@
 //
 // *cnf-optimize-data-performance-acc100.adoc
 
-[id="installing-the-sriov-fec-operator_{context}"]
+[id="cnf-installing-the-sriov-fec-operator_{context}"]
 = Installing the OpenNESS SR-IOV Operator for Wireless FEC Accelerators
 
-The role of the OpenNESS Operator for Intel Wireless Forward Error Correction (FEC) Accelerator is to orchestrate and manage the devices exposed by a range of Intel vRAN FEC acceleration hardware within the {product-title} cluster.
+The role of the OpenNESS Operator for Intel Wireless forward error correction (FEC) Accelerator is to orchestrate and manage the devices exposed by a range of Intel vRAN FEC acceleration hardware within the {product-title} cluster.
 
 One of the most compute-intensive 4G/LTE and 5G workloads is RAN layer 1 (L1) forward error correction (FEC). FEC resolves data transmission errors over unreliable or noisy communication channels. FEC technology detects and corrects a limited number of errors in 4G/LTE or 5G data without the need for retransmission.
 
-The FEC device provided by the Intel® vRAN Dedicated Accelerator ACC100 supports the vRAN use case.
+The FEC device provided by the Intel vRAN Dedicated Accelerator ACC100 supports the vRAN use case.
 
-The SR-IOV FEC Operator creates virtual functions (VFs) for the FEC device, binds them to appropriate drivers, and configures the VFs queues for the desired functionality in 4G/LTE or 5G deployment.
+The OpenNESS SR-IOV Operator for Wireless FEC Accelerators provides functionality to create virtual functions (VFs) for the FEC device, binds them to appropriate drivers, and configures the VFs queues for functionality in 4G/LTE or 5G deployment.
 
 As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the {product-title} CLI or the web console.
 
 [id="installing-sriov-fec-operator-using-the-cli_{context}"]
-== Installing the SR-IOV FEC Operator by using the CLI
+== Installing the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI
 
-As a cluster administrator, you can install the Operator by using the CLI.
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI.
 
 .Prerequisites
 
-* A cluster installed on bare metal hardware.
+* A cluster installed on bare-metal hardware.
 * Install the OpenShift CLI (`oc`).
 * Log in as a user with `cluster-admin` privileges.
 
 .Procedure
 
-. Create a namespace for the SR-IOV FEC Operator by completing the following actions:
+. Create a namespace for the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by completing the following actions:
 
 .. Define the `vran-acceleration-operators` namespace by creating a file named `sriov-namespace.yaml` as shown in the following example:
```

Comment:
The letter for for the second sub-step isn't building. Try adding a + between the steps (line 45) to see if that fixes the issue.

Reply:
@bmcelvee added though I can't see the issue with it not building to be honest.